engine/lib/entities.php

```diff
@@ -724,10 +723,6 @@ function elgg_instanceof($entity, $type = null, $subtype = null, $class = null)
 		$return = $return && ($entity->getSubtype() == $subtype);
 	}
 
-	if ($class) {
```

Comment:
Since we didn't deprecate this, what do you think about logging an error if `func_num_args() > 3`?